tests/monster_test_generated.h

```diff
@@ -556,7 +556,7 @@ inline const AnyAmbiguousAliases (&EnumValuesAnyAmbiguousAliases())[4] {
     AnyAmbiguousAliases_NONE,
     AnyAmbiguousAliases_M1,
     AnyAmbiguousAliases_M2,
-    AnyAmbiguousAliases_M3
+    AnyAmbiguousAliases_M3,
```

Comment:
These changes are adding noise, can we remove them from this PR?

Reply:
done.